platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationFileDirectory.java

```diff
@@ -91,7 +110,11 @@ public ConfigurationFileDirectory(@NotNull DirectoryStream directoryStream,
      */
     public void init() throws IOException {
         createDirectory(processedDirectory);
+        LOGGER.info("created processed directory");
```

Comment:
Should we move those `LOGGER` calls inside the `createDirectory()` method and making them `debug` logs instead of `info`?